app/assets/stylesheets/blacklight/_search_results.scss

```diff
@@ -0,0 +1,28 @@
+// the search results on the index view of the catalog
+#documents {
+  .document {
+    margin-top: $padding-base-vertical;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered border-bottom, margin-top, padding-top